types/splitpanes/index.d.ts

```diff
@@ -1,85 +1,191 @@
+    onSplitterDblClick: (event: Event, splitterIndex: number) => Pane | void;
+    onPaneClick: (event: Event, splitterIndex: number) => void;
+    getCurrentMouseDrag: (event: Event) => CoordinateType;
+    getCurrentDragPercentage: (drag: CoordinateType) => number;
+    calculatePanesSize: (drag: CoordinateType) => void;
+    doPushOtherPanes: (sums: SumsType, dragPercent: number) => null | {};
+    sumPrevPanesSize: (splitterIndex: number) => number;
+    sumNextPanesSize: (splitterIndex: number) => number;
+    findPrevExpandedPane: (splitterIndex: number) => Pane | {};
+    checkSplitpanesNodes: () => void;
+    addSplitter: (paneIndex: number, nextPaneNode: Node, isVeryFirst: boolean) => void;
+    removeSplitter: (node: Node) => void;
+    redoSplitters: () => void;
+    requestUpdate: (arg0: RequestUpdateType) => void;
+    onPaneAdd: (pane: Pane) => void;
+    onPaneRemove: (pane: Pane) => void;
+    resetPaneSizes: (changedPanes: ChangedPaneType) => void;
+    equalize: () => void;
+    initialPanesSizing: () => void;
+    equalizeAfterAddOrRemove: (arg0: ChangedPaneType) => void;
+    readjustSizes: (leftToAllocate: number, ungrowable: number[], unshrinkable: number[]) => void;
 }
 
-export interface splitPanesConstructor extends VueConstructor {
-  props: splitPanesProps;
-  data: () => splitPanesData;
-  watch: splitPanesWatch;
-  methods: splitPanesMethods;
-  computed: splitPanesComputed;
-  mounted: () => void;
-  beforeDestroy: () => void;
-  beforeUpdate: () => void;
-  render: (createEl: any) => any;
+export interface SplitpaneWatch {
+    panes: SplitpaneWatchPaneType;
+    horizontal: () => void;
+    firstSplitter: () => void;
+    dblClickSplitter: (enable: boolean) => void;
 }
+
+export interface Splitpane extends VueConstructor {
+    props: SplitpaneProps;
+    data: () => SplitpaneData;
+    computed: SplitpaneComputed;
+    methods: SplitPaneMethods;
+    watch: SplitpaneWatch;
+    mounted: () => void;
+    beforeDestroy: () => void;
+    beforeUpdate: () => void;
+    render: (createEl: any) => any;
+}
+
+export interface PaneProps {
+    size: number | string;
+    minSize: number | string;
+    maxSize: number | string;
+}
+
+export interface PaneData {
+    style: any;
+}
+
+export interface PaneMethods {
+    update: (style: any) => void;
+}
+
+export interface PaneComputed {
+    sizeNumber: () => number | null;
+    maxSizeNumber: () => number;
+    minSizeNumber: () => number;
+}
+
+export interface PaneWatch {
+    sizeNumber: (size: number) => void;
+    minSizeNumber: (min: number) => void;
+    maxSizeNumber: (max: number) => void;
+}
+
+export interface Pane extends VueConstructor {
+    props: PaneProps;
+    data: () => PaneData;
+    methods: PaneMethods;
+    computed: PaneComputed;
+    watch: PaneWatch;
+    mounted: () => void;
+    beforeDestroy: () => void;
+    beforeUpdate: () => void;
+    render: (createEl: any) => any;
+}
+
+export const Pane: Pane;
+export const Splitpane: Splitpane;
```

Comment:
The export is named `Splitpanes`. See https://unpkg.com/browse/splitpanes@2.3.6/src/components/splitpanes/index.js.

Reply:
Updated!